src/App.js

```diff
@@ -1,26 +1,104 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import 'bulma/css/bulma.css';
+import foods from './foods.json';
+import FoodBox from './components/FoodBox';
+import AddFood from './components/AddFood';
+import Search from './components/Search'
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends React.Component {
+
+
+  state = {
+    foods: foods,
+    filteredFoods: foods,
+    todayFoods: [],
+    wantsToAdd: false
+  }
+
+  handleAddFood = (event) => {
+    event.preventDefault()
+    let foodName = event.currentTarget.food.value
+    let foodCal = event.currentTarget.calories.value
+    let foodImg = event.currentTarget.image.value
+    let newFoods = [{name: foodName, calories: foodCal, image: foodImg}, ...this.state.foods]
+    this.setState({
+      foods: newFoods,
```

Comment:
here you are updating the foods state but not the filteredFoods state. In L76 you are iterating over the filderedFoods, so the added items will only appear when you are also using the search feature.